Automattic/sensei
# Add AutomateWoo course actions

This PR adds 2 new AutomateWoo course actions.

## Proposed Changes

* A new action for adding a student to a course.
* A new action for removing a student from a course.

## Testing Instructions


1. Run `composer install` to update the autoloader.
1. Install the AutomateWoo plugin.
1. Go to AutomateWoo -> Workflows.
1. Create a new workflow. For the trigger select "Student Added To the Group" (you may choose another if you don't have Sensei Pro). For the action use "Add to Course" and select a few courses.
1. Create a new workflow. For the trigger select "Student Removed From the Group" (you may choose another if you don't have Sensei Pro). For the action use "Remove from Course" and select the same courses.
1. Go to Sensei LMS -> Groups and add a user to a group.
1. Make sure the user is also added to the selected courses by the "Add to Course" action.
1. Now remove the same user from a group and make sure the user is removed from the selected courses by the "Remove from Course" action.

I was not able to add tests for the AutomateWoo actions because they are extending an abstraction class provided by the AutomateWoo plugin.

## Pre-Merge Checklist
